### ISSUE TITLE
ffmpeg fails to add thumbnail when using the --embed-thumbnail option on tracks with umlauts in the title

### ISSUE THREAD

**Issue by christxph** [rg3/youtube-dl#10710]
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2016.09.19**
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)

---

```
$ youtube-dl -v --embed-thumbnail https://soundcloud.com/chris-meid/chris-meid-red-river-ft-tyler-s
jostrom-martin-wagler
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-v', u'--embed-thumbnail', u'https://soundcloud.com/chris-meid/chris-meid-red-river-ft-tyler-sjostrom-martin-wagler']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.09.19
[debug] Python version 2.7.10 - CYGWIN_NT-6.2-2.4.1-0.293-5-3-x86_64-64bit
[debug] exe versions: ffmpeg N-76684-g1fe82ab, ffprobe N-76684-g1fe82ab
[debug] Proxy map: {}
[soundcloud] chris-meid/chris-meid-red-river-ft-tyler-sjostrom-martin-wagler: Resolving id
[soundcloud] chris-meid/chris-meid-red-river-ft-tyler-sjostrom-martin-wagler: Downloading info JSON
[soundcloud] 223355094: Downloading track url
[soundcloud] 223355094: Checking fallback video format URL
[soundcloud] 223355094: Checking http_mp3_128_url video format URL
[soundcloud] 223355094: Downloading thumbnail ...
[soundcloud] 223355094: Writing thumbnail to: CHRIS MEID - Red River (ft. Tyler Sjöström & Martin Wagler)-223355094.jpg
[debug] Invoking downloader on u'https://cf-media.sndcdn.com/bREiwdSABDK9.128.mp3?Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiKjovL2NmLW1lZGlhLnNuZGNkbi5jb20vYlJFaXdkU0FCREs5LjEyOC5tcDMiLCJDb25kaXRpb24
iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjE0NzQzNzc3MzN9fX1dfQ__&Signature=dwm5thk1dumfrywIYM1In1FmLc-sdAyLqD80suhkgFRaGFX0br9Vk3WRSl9he2HnS7FIQuEug9mv6JcDtNir2aGuqn24gyO1zLOC8SycPxLyEEmKk4T9UjYN
86a33DwoRpedaygSvY2CPHtgO-hrao7esFAMr6w1TQxhGhoUovRmVJlxTK0XZvkiStGLgl1WrlMcwblLfCj70pZ8fBBC984SWaNIoRTxmPa~Gd3zcDRpNPASaiN-0MNwsEyTZOeCzukc2ZxXp1fp~53BFpraCTSJqg9ptg0N6A4~9MOV9tjmzWOR9gC3n~y7D2SJnz45
wO-CIaZlfE-HqvHCeUS1rw__&Key-Pair-Id=APKAJAGZ7VMH2PFPW6UQ'
[download] CHRIS MEID - Red River (ft. Tyler Sjöström & Martin Wagler)-223355094.mp3 has already been downloaded
[download] 100% of 2.67MiB
[ffmpeg] Adding thumbnail to "CHRIS MEID - Red River (ft. Tyler Sjöström & Martin Wagler)-223355094.mp3"
[debug] ffmpeg command line: ffmpeg -y -i 'file:CHRIS MEID - Red River (ft. Tyler Sjöström & Martin Wagler)-223355094.mp3' -i 'file:CHRIS MEID - Red River (ft. Tyler Sjöström & Martin Wagler)-22335509
4.jpg' -c copy -map 0 -map 1 -metadata:s:v 'title="Album cover"' -metadata:s:v 'comment="Cover (Front)"' 'file:CHRIS MEID - Red River (ft. Tyler Sjöström & Martin Wagler)-223355094.temp.mp3'
ERROR: file:CHRIS MEID - Red River (ft. Tyler Sjöström & Martin Wagler)-223355094.jpg: No such file or directory
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1837, in post_process
    files_to_delete, info = pp.run(info)
  File "/usr/local/bin/youtube-dl/youtube_dl/postprocessor/embedthumbnail.py", line 50, in run
    self.run_ffmpeg_multiple_files([filename, thumbnail_filename], temp_filename, options)
  File "/usr/local/bin/youtube-dl/youtube_dl/postprocessor/ffmpeg.py", line 192, in run_ffmpeg_multiple_files
    raise FFmpegPostProcessorError(msg)
FFmpegPostProcessorError

```

---

When downloading a track off soundcloud with an umlaut in its title (e.g. äöü) and the flag --embed-thumbnail ffmpeg fails to add the thumbnail to the track and exits with the error 'No such file or directory' even though the thumbnail has been downloaded successfully by youtube-dl.

Tested on win8 with cygwin 

Example urls: https://soundcloud.com/gunesergun/chris-meid-red-river-ft-tyler-sjostrom-martin-waglergunes-ergun-remix and https://soundcloud.com/achtabahn/korner-absturzen-achtabahn-mix

Related issues: https://github.com/rg3/youtube-dl/pull/5585 https://github.com/rg3/youtube-dl/issues/5527 and https://github.com/rg3/youtube-dl/issues/4787

**Comment by yan12125** [comment]
Does it work for songs without umlauts?

**Comment by yan12125** [comment]
By the way, this looks like a duplicate of #7238. Please consider updating ffmpeg.

**Comment by christxph** [comment]
> Does it work for songs without umlauts?
It does, that's why I specifically mentioned that the problem only occurs when downloading tracks with umlauts in their title.

> Please consider updating ffmpeg.
I have version N-76684-g1fe82ab installed at the moment, will look for an update now.

**Comment by christxph** [comment]
Update: Fixed by updating ffmpeg to the latest build 11777eb (2016-09-19), sorry for not thinking about that beforehand.

**Comment by yan12125** [comment]
No problem. Glad to see that fixed!
